resources/views/community/components/event/aotw.blade.php

```diff
@@ -1,31 +1,49 @@
+@props([
+    'achievement' => [],
+    'game' => [],
+    'consoleName' => '',
+    'forumTopicId' => 0,
+])
+
 <?php
+$achievementId = $achievement->ID;
+$achievementName = $achievement->Title;
+$achievementDescription = $achievement->Description;
+$achievementPoints = $achievement->Points;
+$achievementRetroPoints = $achievement->TrueRatio;
+$achievementBadgeName = $achievement->BadgeName;
 
-use App\Site\Models\StaticData;
+$renderedGameTitle = renderGameTitle($game->Title);
+$achievementIconSrc = media_asset("/Badge/$achievementBadgeName.png");
+$gameSystemIconUrl = getSystemIconUrl($game->ConsoleID);
+?>
 
-$staticData = StaticData::first();
+<div class="component">
+    <h3>Achievement of the Week</h3>
 
-if ($staticData === null) {
-    return;
-}
+    <div class="bg-embed p-4 rounded border border-embed-highlight">
+        <a href="{{ route('achievement.show', $achievement->ID) }}" class="group flex gap-x-2 mb-3 text-text transition-transform hover:-translate-y-0.5">
+            <img src="{{ $achievementIconSrc }}" alt="Achievement of the week badge" width="64" height="64" class="w-16 h-16">
+            <div>
+                <p class="font-semibold leading-4 text-link group-hover:text-link-hover">{{ $achievement->Title }}</p>
```

Comment:
Do we want to process tags on the title?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/5bb97094-c527-4348-951e-13a3ad78222f)

Reply:
We do indeed, good catch. This should also be addressed now.